### ISSUE TITLE
Padlock function ? (Update screen shot and read me file)

### ISSUE THREAD

**Issue by vdoucet** [SergiuToporjinschi/node-red-contrib-heater-controller#17]
Hi and thank you for the great work.
I'm switching my home automation system to node-red and I'm using your thermostat (ver. 1.2.0). So far it runs successfuly but I have a question regarding the padlock function on the left of the UI.
What is it supposed to do ? I did some tests but I didn't understand the role.
If you could give some hints ...
Thx,
Vincent

**Comment by SergiuToporjinschi** [comment]
Hi,
Once you set a target value, that will be available only until next change in calendar, if the padlock is unlocked;
That means you are saying something like I want the new target until next change, no matter if the next value from calendar is less then target or greater then target. 
Let's say is 3pm and the calendar say that should be 20 degrees until 5pm when the new temperature will be 23 degrees. And you arrive early at home, only today, and you will find the thermostat on 20 degree, how is scheduled. You want to have 23 degrees but you want that only at 5pm when the schedule is correct 23. Then you have to change only the target.
If you lock it then will keep the target value without any limit, no matter what the schedule is. 
So, on previous scenario, if you set 24 degrees and lock it then will keep 24 until you are changing back.
Long story short, if is locked then it will keep that unlimited time, until you unlock it.

**Comment by vdoucet** [comment]
Thanks Sergiu, it's clear !
